P/Polyline/Compat.toml

```diff
@@ -0,0 +1,2 @@
+[0]
+julia = "1.1-1"
```

Comment:
Doesn't work on Julia 1.0?

Reply:
It should. I can add the tests to travis. Is that typically the current lower bound you would go to? Or it  might be useful to have 0.7 as well?

Reply:
Most packages are 1.0 and up now.

Reply:
Thanks. I have updated it to start from v1.0